src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -180,6 +180,11 @@
     protected int defaultAssemblyRegionPadding() { return 100; }
 
     @Override
+    protected int defaultMaxReadsPerAlignmentStart() {
+        return 50;
```

Comment:
just nitpicking, but the vertical whitespacing here is different from the other nearby methods